src/main/java/fr/sncf/osrd/cli/SimulateCommand.java

```diff
@@ -70,6 +71,9 @@ public int run() {
             for (var trainSchedule : config.trainSchedules)
                 TrainCreatedEvent.plan(sim, trainSchedule);
 
+            // initialize the switch post with the train succession tables
+            sim.infraState.switchPost.init(config.infra, config.switchSuccessions);
```

Comment:
This should not be done here, the initialization of the infra should be internal. We run simulations in other places too.

The tricky thing will be to forward the succession config up to the switchPost constructor. Maybe we could include it in the config.infra somehow?